### PR TITLE
Support hashClientId

### PR DESCRIPTION
**Pull request by j450h1**
See PR https://github.com/MarkEdmondson1234/googleAnalyticsR/issues/191

### CODE REVIEW

File: R/client_id.R [hunk 1]
```diff
@@ -0,0 +1,26 @@
+#' Get hashed version of client id (also known as hashClientId, hashedClientId, or BigQuery's fullVisitorId)
+#' 
+#' @param webPropertyId Web Property Id
+#' @param clientId Client Id
+#'
+#' @return hashedClientId string
+#' @importFrom googleAuthR gar_api_generator
+#' @family managementAPI functions
+#' @export
+hashed_client_id <- function(webPropertyId,
```
Comment (MarkEdmondson1234) [inline R/client_id.R]: For autocomplete help, and to make it easier to vectorise over, I suggest:

```r
ga_clientid_hash <- function(clientId, webPropertyId){
   ...etc...
```

Comment (j450h1) [reply R/client_id.R]: Of course, makes sense. I will make update shortly. I was debating whether I should return only the hashed value as a string or the entire object which also includes the original clientid as well as some other data (this is current behaviour). Any thoughts on that?

Comment (MarkEdmondson1234) [reply R/client_id.R]: I think the entire object as you never know if people will have use of it.  It shouldn't take too much data processing to output just the clientId.

**Comment by MarkEdmondson1234** [comment]
Great thanks! Closes #191

**Pull request merged**